code/modules/projectiles/guns/ballistic/rifle.dm

```diff
@@ -87,6 +87,35 @@
 	fire_sound = 'sound/weapons/gun/sniper/shot.ogg'
 	can_be_sawn_off = FALSE
 
+/obj/item/gun/ballistic/rifle/boltaction/musket
+	name = "maintenance musket"
+	desc = "An excellent weapon for flushing out tunnel rats and enemy assistants, but its rifling leaves much to be desired."
+	icon_state = "musket"
+	inhand_icon_state = "musket"
+	worn_icon_state = "musket"
+	fire_sound = 'sound/weapons/gun/sniper/shot.ogg'
+	mag_type = /obj/item/ammo_box/magazine/internal/boltaction/musket
+	initial_calibur = "shotgun"
+	alternative_calibur = "a762"
+	initial_fire_sound = 'sound/weapons/gun/sniper/shot.ogg'
+	alternative_fire_sound = 'sound/weapons/gun/shotgun/shot.ogg'
+	can_modify_ammo = TRUE
+	can_misfire = TRUE
+	misfire_probability = 10
+	can_bayonet = FALSE
+	can_be_sawn_off = FALSE
+	projectile_damage_multiplier = 0.75
```

Comment:
if the basic musket has too high of a DPS, why not just increase the (or add a) cooldown for racking it slightly to decrease its effective rate of fire? would the up front damage still be too high to be "balanced"?

Reply:
Well no the problem is mostly that it's hard to provide good feedback on the racking part and it's gonna feel like arse to slow that down. It's easy to misinterpret that as actual game lag.

I still think there is merit to circumventing the damage output a little by being quick on the clicks, rather than slowing down click rate artificially and potentially making you think you are lagging out or some shit.